pkgs/development/libraries/nvidia-texture-tools/default.nix

```diff
@@ -26,6 +19,9 @@ stdenv.mkDerivation rec {
     # Make a recently added pure virtual function just virtual,
     # to keep compatibility.
     sed -i 's/virtual void endImage() = 0;/virtual void endImage() {}/' src/nvtt/nvtt.h
+    # remove x86_64-only libraries
```

Comment:
Can we do this conditonally on aarch64?

Reply:
Conditional paching can be quite error prone, so I'd be reluctant to do it, unless it breaks other platforms.